src/components/04-pages/partners/organization/experiences/experiences.tsx

```diff
@@ -0,0 +1,40 @@
+import { Typography, useTheme } from '@mui/material';
+import Breadcrumbs from '../../../../01-elements/breadcrumbs/breadcrumbs';
+import DataTableExperiences, { ExperienceRow } from '../../../../02-components/data-table/data-table-experiences';
+
+export type PartnerExperiencePageProps = {
```

Comment:
We need to define separate variables this Component accepts instead of passing an entire object